x-pack/plugins/apm/server/routes/register_routes/index.ts

```diff
@@ -29,6 +29,13 @@ const inspectRt = t.exact(
   })
 );
 
+const CLIENT_CLOSED_REQUEST = {
+  statusCode: 499,
+  body: {
+    message: 'Client closed request',
+  },
+};
```

Comment:
Not a big deal but if possible it would be better to use the official error codes that are part of Boom.

499 is only implemented as "client closed" in nginx afaict:
![image](https://user-images.githubusercontent.com/209966/125068391-b7a9e800-e0b5-11eb-9673-6c936eb0542a.png)

Other servers implement it differently:
![image](https://user-images.githubusercontent.com/209966/125068428-c1cbe680-e0b5-11eb-969b-20434edc4ad7.png)

Reply:
I'm honestly not sure why it's there (I didn't change it). any idea for a better one?

Reply:
I looked and couldn't find any good fit. So I'll leave it up to you if you can find something better or just leave it.

Reply:
will leave it then. it's a status code the browser will never see anyway.